test/qt3tests.ts

```diff
@@ -835,5 +1056,20 @@ describe('qt3 test set', () => {
 		// tslint:disable-next-line: no-console
 		console.log(`Writing a log of ${expressionBackendLog.length}`);
 		testFs.writeFileSync('unrunnableTestCases.csv', expressionBackendLog.join('\n'));
+		if (reportJSCodeGen) {
+			let report = 'succesFullCases: \n\n';
```

Comment:
I think it's successFul, single L. https://whichiscorrect.com/successful-or-successfull/